Dockerfile

```diff
@@ -115,7 +115,8 @@ HEALTHCHECK --interval=5m --timeout=5s CMD nc -z 127.0.0.1 4822 || exit 1
 
 # Create a new user guacd
 ARG UID=1000
-RUN useradd --system --create-home --shell /usr/sbin/nologin --uid $UID --no-user-group guacd
+RUN groupadd --gid $UID guacd
+RUN useradd --system --create-home --shell /usr/sbin/nologin --uid $UID --gid $UID guacd
```

Comment:
I don't think the GID should be set to the UID (or that it should be assumed that doing so is safe). Perhaps this should be its own `ARG`?

Reply:
Many thanks for your feedback @mike-jumper, updated accordingly :+1:

Reply:
Note that there's no risk for both `UID` and `GID` to have the same numerical value (`1000` here).

Reply:
Well ... there is no risk if they _happen_ to have the same value, or if the same value is intentionally chosen for both the UID and the GID after that value was been confirmed to be available within both UID and GID pools.

The important thing is to not have the image build written to directly use a UID as if it were a GID.

Reply:
OK, both UID and GID with a `1000` value are free for use in the image (the free range around 1000 is rather large).
And both are now identified with their own `ARG`.
Should then be OK, or did I perhaps miss something ?
Thank you again 👍